### DISCUSSION TITLE
Copy all to..

### DISCUSSION THREAD

**Discussion by Criscia68** [WinMerge/winmerge#2176] [Ideas]
With three files open ( left - middle - right ) I can't copy everything from the right to the left

**Comment by SommenJeroen** [comment]
Still does not work

**Reply by sdottaka** [reply]
Please activate the right pane, press Ctrl + A to select all items, then right-click and choose "Copy to Left" to copy everything to the left side.

<img width="1919" height="468" alt="image" src="https://github.com/user-attachments/assets/ad6e693a-646e-4762-916f-24410908d381" />

**Reply by SommenJeroen** [reply]
Does not work with directories
<img width="2560" height="1400" alt="Schermafbeelding 2026-04-16 132841" src="https://github.com/user-attachments/assets/365bebeb-76fc-494c-929e-38643f6a8469" />

**Reply by sdottaka** [reply]
Select the files you want to copy, then right-click and choose "Right to Left".

<img width="897" height="885" alt="image" src="https://github.com/user-attachments/assets/a2b97dba-0ba1-4c08-a5e8-f46df07d7948" />

**Reply by SommenJeroen** [reply]
That does not copy them to both (middle and left), only to the left...

**Reply by sdottaka** [reply]
Unfortunately, there is no feature to copy from the right pane to both the middle and left panes at the same time.

**Reply by SommenJeroen** [reply]
Thank U for answering, It would be a nice feature I think to be able to copy from one pane to other panes at the same time
Where to suggest new features?
